src/actions/eventActions.js

```diff
@@ -23,25 +26,82 @@ export const fetchEvent = (incidentId, eventId) => reduxBackedPromise(
     getEventActionSet(incidentId, eventId)
 )
 
-export const fetchEvents = (incidentId) => reduxBackedPromise(
-    [(incidentId ? 'incidents/' + incidentId + '/': '') + 'events/'],
-    getEventsActionSet(incidentId)
-)
+export const fetchEvents = (filter) => reduxBackedPromise(
+        getEventsFetchArgs(filter),
```

Comment:
Since this builds a URL that isn't compatible with the current version of the Gateway, we will need to update the Gateway before we merge this pull request.

Reply:
Gateway fixes are complete and need to be merged.  This item now has master's auth changes folded in (and has been adjusted to work w/those changes).  It also now clears all events that are not wanted (not filtered for) and pulls autocomplete choices from the eventTypes in state. @pdimitratos  @maggiepint

Reply:
Gateway is merged.  This is no longer an issue.